sig-autoscaling/annual-report-2021.md

```diff
@@ -23,60 +22,63 @@ In future, this will be generated from kubernetes/enhancements kep.yaml files
-     - [$kep-number - $title](https://git.k8s.io/community/$link/README.md) - $milestone.alpha
-     - [$kep-number - $title](https://git.k8s.io/community/$link/README.md) - $milestone.alpha
-   - Pre-alpha
-     - [$kep-number - $title](https://git.k8s.io/community/$link/README.md)
+     - [2702 - Graduate v2beta2 Autoscaling API to GA](https://git.k8s.io/enhancements/keps/sig-autoscaling/2702-graduate-hpa-api-to-GA/README.md) - $milestone.stable
+
+   - Subproject enhancements:
+     - [Support Customized Recommenders for Vertical Pod Autoscalers](https://github.com/kubernetes/autoscaler/tree/master/vertical-pod-autoscaler/enhancements/3919-customized-recommender-vpa)
+     - [MinReplicas per VPA object](https://github.com/kubernetes/autoscaler/tree/master/vertical-pod-autoscaler/enhancements/4566-min-replicas#kep-4566-minreplicas-per-vpa-object)
+     - [https://github.com/kubernetes/autoscaler/blob/68c984472acce69cba89d96d724d25b3c78fc4a0/cluster-autoscaler/proposals/plugable-provider-grpc.md](https://github.com/kubernetes/autoscaler/blob/68c984472acce69cba89d96d724d25b3c78fc4a0/cluster-autoscaler/proposals/plugable-provider-grpc.md)
+     - [Expander Plugin over gRPC](https://github.com/kubernetes/autoscaler/blob/66af6d1339f86e87a37d5f505109b59c729de198/cluster-autoscaler/proposals/expander-plugin-grpc.md)
 
 ## Project health
 
 1. What areas and/or subprojects does your group need the most help with?
    Any areas with 2 or fewer OWNERs? (link to more details)
 
-   -
-   -
-   -
+   - Code contributors and reviewers
+   - Issue triage/response
+   - Whilst there are no areas with explicitly 2 or few owners, there are a number of areas where we have this few subject matter experts, acting as a significant bottleneck for significant changes
 
 2. What metrics/community health stats does your group care about and/or measure?
 
-   -
-   -
-   -
+   - Currently lacking in measurement of community health stats
+   - Care about growing number of contributors
+   - Would like to begin measuring time to merge of community PRs
 
 3. Does your [CONTRIBUTING.md] help **new** contributors engage with your group specifically by pointing
    to activities or programs that provide useful context or allow easy participation?
 
-   -
+   - Not currently
 
 4. If your group has special training, requirements for reviewers/approvers, or processes beyond the general [contributor guide],
    does your [CONTRIBUTING.md] document those to help **existing** contributors grow throughout the [contributor ladder]?
 
-   -
+   - N/A
 
 5. Does the group have contributors from multiple companies/affiliations?
 
-   -
+   - Yes - though the core approvers are almost all from a single company.
 
 6. Are there ways end users/companies can contribute that they currently are not?
    If one of those ways is more full time support, what would they work on and why?
 
-   -
-   -
+   - Increased support for some cloud provider implementations for the Cluster Autoscaler
```

Comment:
can you include links here that are actionable for end users who are reading the report?